.obs/dockerfile/teal-iso/Dockerfile

```diff
@@ -1,12 +1,14 @@
 # SPDX-License-Identifier: Apache-2.0
-#!BuildTag: rancher/elemental-teal-iso/5.3:latest
-#!BuildTag: rancher/elemental-teal-iso/5.3:%VERSION%
-#!BuildTag: rancher/elemental-teal-iso/5.3:%VERSION%-%RELEASE%
+#!BuildTag: rancher/elemental-teal-iso:latest
```

Comment:
Hmm, this might fail while we build two versions (like `5.3` and `5.4`) in parallel.

We should probably just drop the `:latest` tag ?!

Reply:
In OBS builds this will end up being something like:
```
registry.org/home/cassany/elementalnewdev/containers/rancher/elemental-teal-iso:latest
registry.org/home/cassany/elementalnewdev/containers_next/rancher/elemental-teal-iso:latest
```

So there will not be an overlap as long as they build in different repositories. For IBS builds under registry.suse.com I see your point. If we follow other SUSE images approach we could keep it our latest code base stream build and remove it on old ones. Using `suse/sle15:latest` image as an example this shifted from SP4 to SP5 as soon as the new one was released. I agree this is quite annoying from maintenance point of view as this forces us to update the old build in any new release 🤔 also feels like this can be error prone if done manually.